src/modules/catalog.js

```diff
@@ -1787,6 +1798,10 @@ class Catalog {
     const { fetch } = this.options;
     const controller = new AbortController();
     const { signal } = controller;
+    let { matches } = parameters;
+
+    const newParameters = { ...parameters, matches };
+    matches = matches.map((match) => toSnakeCaseKeys(match, false));
```

Comment:
There is a good chance I'm missing something here but I think this gets assigned to `matches` but it doesn't get updated in `newParameters`

Reply:
Great catch, I changed the order of these things at one point for some reason 🤔